main/auth/auth.py

```diff
@@ -132,6 +132,24 @@ def decorated_function(*args, **kwargs):
   return decorated_function
 
 
+def cron_required(f):
+  decorator_order_guard(f, 'auth.cron_required')
+
+  @functools.wraps(f)
+  def decorated_function(*args, **kwargs):
+    if config.PRODUCTION and 'X-Appengine-Cron' in flask.request.headers:
```

Comment:
According to the [docs](https://cloud.google.com/appengine/docs/python/config/cron?hl=en#Python_app_yaml_Securing_URLs_for_cron) yes.. 

>The X-Appengine-Cron header is set internally by Google App Engine. If your request handler finds this header it can trust that the request is a cron request. If the header is present in an external user request to your app, it is stripped, except for requests from logged in administrators of the application, who are allowed to set the header for testing purposes.

> Google App Engine issues Cron requests from the IP address 0.1.0.1.